profiles/blackkspydo.json

```
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/58302072/193435002-246a32fb-714d-4a0f-9674-38eb962ba2f3.png",
+    "issueId": 1437,
+    "name": "Kishor U",
+    "username": "blackkspydo"
+  }
+  
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```